signingscript/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN groupadd --gid 10001 app && \
     useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
 
 RUN apt-get update \
- && apt-get install -y zipalign default-jdk-headless osslsigncode \
+ && apt-get install -y zipalign osslsigncode \
```

Comment:
Hm, did we need this for jar signing before, and no longer?

Reply:
I think maybe we used it for zipalign before it was available as its own package?